src/components/asset-list/RecyclerAssetList2/WrappedCollectiblesHeader.tsx

```diff
@@ -34,7 +35,9 @@ const getMenuItemIcon = (value: NftCollectionSortCriterion) => {
 };
 
 const CollectiblesHeader = () => {
-  const { nftSort, nftSortDirection, updateNFTSort } = useNftSort();
+  const { accountAddress } = useAccountSettings();
+  const { getNftSort, updateNftSort } = useNftSortStore();
+  const nftSort = getNftSort(accountAddress);
```

Comment:
#nitpick but you could destructure the response here and avoid most (if not all) the changes beneath this
```suggestion
  const { sortBy, sortDirection } = getNftSort(accountAddress);
```

Reply:
I guess `nftSort` -> `sortBy` would be necessary still. But yeah this is my brain ultra nitpicking this